index.html

```diff
@@ -2,98 +2,226 @@
+                    sign-ups required.
+                </p>
+                <ul class="categories hex">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javaScript</li>
+                </ul>
+                <div class="see-project">
+                    <a href="#"><button type="button" class="hex">See Project</button></a>
+                </div>
+            </div>
+        </section>
+        <section class="grid-items other-projects">
+            <img class="project-img" src="images/projects/project-4.png" alt="Project-1" />
+            <div class="details">
+                <div class="titles">
+                    <h3 class="project-title">Multi-Post Stories</h3>
+                    <ul class="canopy">
+                        <li>CANOPY</li>
+                        <li class="bullets"><span class="dot"></span></li>
+                        <li class="o-li">Back End Dev</li>
+                        <li class="bullets"><span class="dot"></span></li>
+                        <li class="o-li">2015</li>
+                    </ul>
+                </div>
+                <p>
+                    A daily selection of privately personalized reads; no accounts or
+                    sign-ups required.
+                </p>
+                <ul class="categories hex">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javaScript</li>
+                </ul>
+                <div class="see-project">
+                    <a href="#"><button type="button" class="hex">See Project</button></a>
+                </div>
+            </div>
+        </section>
+    </section>
+    <!-- About myself -->
+    <section class="about-myself">
+        <h2 class="about">About Myself</h2>
+        <p class="text">
+            Hello I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like what
+            you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+        <h2 class="hex connect">LET’S CONNECT</h2>
+        <ul class="social-media">
+            <li clalss="socials">
+                <a href="#"><img src="images/socials/twitter.svg" /></a>
+            </li>
+            <li clalss="socials">
+                <a href="#"><img src="images/socials/Linkedin.svg" /></a>
+            </li>
+            <li clalss="socials">
+                <a href="#"><img src="images/socials/m.svg" /></a>
+            </li>
+            <li clalss="socials">
+                <a href="#"><img src="images/socials/Vector1.svg" /></a>
+            </li>
+            <li clalss="socials">
+                <a href="#"><img src="images/socials/Vector2.svg" /></a>
+            </li>
+        </ul>
+        <div class="see-project">
+            <a href="#"><button type="button" class="hex">Get my resume</button></a>
```

Comment:
- [ ] Please use all Button effects like `hover`, `pressed` etc. check below :point_down: all effects as mentioned in the Figma design

![Screenshot from 2023-02-19 00-56-06](https://user-images.githubusercontent.com/45226848/219902905-47657f84-1edb-4303-b103-d4e7321931ee.png)